R/frequency_encoder.R

```diff
@@ -1,13 +1,18 @@
+
+encode_freq <- function(X) {
+  X
+}
```

Comment:
So what is X here, the data?.

Never mind, here is where you do the magic :)

So I guess we need to put this function below the docstring don't we?

Reply:
Or maybe inside the frequency encoder function so we do not have to write two functions?..